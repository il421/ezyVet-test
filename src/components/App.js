import React from 'react';
import '../styles/main.css';

import Header from './layouts/Header.js';
import Cart from './cart/Cart.js';
import Products from './products/Products.js';

export const App = () => {
  return (
    <div className="app">
      <Header />
      <main className="main">
        <Products />
        <Cart />
      </main>
    </div>
  );
}

export default App;
