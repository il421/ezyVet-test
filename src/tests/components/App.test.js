import React from 'react';
import { shallow } from 'enzyme';
import App from '../../components/App.js';

test('should render App without crashing', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
