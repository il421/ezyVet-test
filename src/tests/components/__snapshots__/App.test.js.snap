// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App without crashing 1`] = `
<div
  className="app"
>
  <App />
  <main
    className="main"
  >
    <Connect(Products) />
    <Connect(Cart) />
  </main>
</div>
`;
